apps/db_assembler/includes/functions.sh

```diff
@@ -301,22 +309,26 @@ function dbasm_db_import() {
 		MYSQL_PASS=$PROMPT_PASS
 	fi
 
+	if [[ -z ${MYSQL_PORT+x} ]]; then
+		MYSQL_PORT=3306
+	fi
+
```

Comment:
thrice!

Reply:
just make a small function and reuse it

Reply:
You are right about the function, done so in the last commit, I also changed it instead of checking if the variable is set it will always set it up to the default port and after it it will eval the configs string and if the port is set overwrite it. This is needed as, if for some reason you char and/or world databse does not have the MYSQL_PORT in the string (they should use default port) and the Auth databse has a non default port it will use the non default port for char and world.